### PR TITLE
Devops 3626 mongodb profiles

### PR DESCRIPTION
**Pull request by wanix**
Common work with @jmlrt to add profiling for MongoDB and being able to create a new replicaset with MongoDB 3.4.

Tested on talend-cloud-integration-tds-mongodb

Local tests:
``` bash
./scripts/local_dev_tests.sh -t role-mongodb-tds-profile-centos-73
...
Finished in 11.61 seconds (files took 0.31261 seconds to load)
131 examples, 0 failures

       Finished verifying <role-mongodb-tds-profile-centos-73> (0m12.31s).
-----> Destroying <role-mongodb-tds-profile-centos-73>...
       ==> default: Forcing shutdown of VM...
       ==> default: Destroying VM and associated drives...
       Vagrant instance <role-mongodb-tds-profile-centos-73> destroyed.
       Finished destroying <role-mongodb-tds-profile-centos-73> (0m3.31s).
       Finished testing <role-mongodb-tds-profile-centos-73> (7m35.24s).
-----> Kitchen is finished. (7m35.44s)
```

``` bash
./scripts/local_dev_tests.sh -t role-mongodb-centos-73
...
Finished in 12.04 seconds (files took 0.3358 seconds to load)
133 examples, 0 failures

       Finished verifying <role-mongodb-centos-73> (0m12.78s).
-----> Destroying <role-mongodb-centos-73>...
       ==> default: Forcing shutdown of VM...
       ==> default: Destroying VM and associated drives...
       Vagrant instance <role-mongodb-centos-73> destroyed.
       Finished destroying <role-mongodb-centos-73> (0m3.34s).
       Finished testing <role-mongodb-centos-73> (7m44.07s).
-----> Kitchen is finished. (7m44.25s)
```

### CODE REVIEW

File: hieradata/environment/vagrant.yaml [hunk 1]
```diff
@@ -7,6 +7,11 @@ tic::services::rt_flow_instance_profile: 'test-instance_profile'
 tic::services::rt_flow_t_release:         'my-release'
 tic::services::rt_flow_t_branch:          'my-branch'
 
+mongodb_tds_profile_overided:
```
Comment (jmlrt) [inline hieradata/environment/vagrant.yaml]: overided should be replaced by overrode

File: site/profile/manifests/mongodb.pp [hunk 2]
```diff
@@ -192,7 +220,9 @@
     syslog         => true,
     create_admin   => $create_admin,
     admin_username => $admin_user,
-    admin_password => $admin_password
+    admin_password => $admin_password,
+    storage_engine => $storage_engine,
+    store_creds    => true,
```
Comment (jmlrt) [inline site/profile/manifests/mongodb.pp]: where is it used?

Comment (wanix) [reply site/profile/manifests/mongodb.pp]: for store_creds ?
it is used for keeping /root/.mongodbrc.js with superadmin credentials and facilitate the mongodb module usage (and also avoid unwanted service restart)

File: hieradata/environment/vagrant.yaml (hunk 1 above)
Comment (wanix) [reply hieradata/environment/vagrant.yaml]: I'll fix it

**Pull request merged**